packages/frontend/src/components/ActionPanel/actionPanel.module.css

```diff
@@ -51,6 +52,7 @@
   right: 0;
   display: flex;
   justify-content: center;
+  z-index: 100;
```

Comment:
hvorfor 100?

Reply:
Helt random for å få den på topp. Har vi noen god regel for hvordan å organisere z-index?

Reply:
Vi burde lage en oversikt - lager en backlogoppgave på det